superset/sql_parse.py

```diff
@@ -458,3 +460,178 @@ def validate_filter_clause(clause: str) -> None:
                 )
     if open_parens > 0:
         raise QueryClauseValidationException("Unclosed parenthesis in filter clause")
+
+
+def has_table_query(statement: Statement) -> bool:
```

Comment:
@betodealmeida there's also [this](https://github.com/andialbrecht/sqlparse/blob/master/examples/extract_table_names.py) example which has logic for identifying tables.

Reply:
I don't remember the details, but I've had issues with that example code before — I think it failed to identify table names when they were considered keywords (even though the example calls it out).